stdlib/public/core/Integers.swift

```diff
@@ -23,6 +23,80 @@ extension ExpressibleByIntegerLiteral
   }
 }
 
+//===----------------------------------------------------------------------===//
+//===--- AdditiveArithmetic -----------------------------------------------===//
+//===----------------------------------------------------------------------===//
+
+public protocol AdditiveArithmetic : Equatable {
+  /// The zero value.
```

Comment:
Can we expand this comment with a note that `zero` is the name of the additive identity? Something like
/// zero is the identity element for addition; for any value x,`x + .zero == x` and `.zero + x == x`.

Reply:
Done. There's still no doc comment on the protocol itself, but writing good doc comments would takes time :) I added `TODO`'s for that.